tests/api/plugins/i18n/content-api/content-api.test.api.js

```diff
@@ -187,18 +171,5 @@ describe('i18n - Content API', () => {
       expect(statusCode).toBe(200);
       expect(body.data).toMatchObject(transformToRESTResource(data.homepages[1]));
     });
-
-    // TODO V5: Decide if we want to support locale=all
```

Comment:
Removed this and the previous test because the answer to the comment is "no, only for certain methods like delete"